udata/core/badges/api.py

```diff
@@ -3,8 +3,43 @@
 
 from udata.api import api, fields
 
+from .forms import badge_form
+
+
 badge_fields = api.model('Badge', {
     'kind': fields.String(description=('Kind of badge (certified, etc), '
                                        'specific to each model'),
                           required=True),
 })
+
+
+def add_badge_api(obj):
```

Comment:
I like the factory pattern but I will do like this, to be explicit about the fact it's meant for the API
```python
#badges/api.py
def add():
    pass

def remove():
    pass

#somewhere/somemodule.py
# from udata.features import badges (futur)
from udata.core import badges
badges.api.add()
```
Or other alternative: 
```python
from udata.core.badges import api as badge_api
badges_api.add()
```

I have to check if when importing badges in other modules not needing API there is no side effects.